ci-operator/config/openshift-kni/eapol-operator/openshift-kni-eapol-operator-main.yaml

```diff
@@ -0,0 +1,62 @@
+base_images:
+  authenticator-base:
+    name: "4.14"
+    namespace: ocp
+    tag: base-rhel9
+  builder:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.20-openshift-4.14
+  operator-base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
+build_root:
+  project_image:
+    dockerfile_path: images/build-root/Dockerfile
+images:
+- dockerfile_path: Dockerfile
+  from: operator-base
+  inputs:
+    builder:
+      as:
+      - golang:1.20
+  to: eapol-operator
+- dockerfile_path: Dockerfile.authenticator
+  from: authenticator-base
+  inputs:
+    builder:
+      as:
+      - docker.io/golang:1.20.2
```

Comment:
I would try to sync the golang versions, so theres only one for the whole operator.